### PR TITLE
[bugfix](multi-catalog) Esexternalcatalog is missing LastUpdateTime.

### LINKED ISSUE #24558: [Bug] (multi-catalog) Esexternalcatalog is missing LastUpdateTime.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Version

master

### What's Wrong?

image

### What You Expected?

Show LastUpdateTime.

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by liugddx**
## Proposed changes

Issue Number: close #24558

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
@BePPPower PTAL

**Comment by liugddx** [comment]
@qidaye PTAL

File: fe/fe-core/src/main/java/org/apache/doris/datasource/EsExternalCatalog.java [hunk 1]
```diff
@@ -125,26 +122,6 @@ protected void initLocalObjectsImpl() {
         }
     }
 
-    @Override
```
Comment (qidaye) [inline fe/fe-core/src/main/java/org/apache/doris/datasource/EsExternalCatalog.java:128]: Why remove this?

File: fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalCatalog.java [hunk 2]
```diff
@@ -279,8 +279,7 @@ protected void init() {
         }
         dbNameToId = tmpDbNameToId;
         idToDb = tmpIdToDb;
-        long currentTime = System.currentTimeMillis();
-        lastUpdateTime = currentTime;
+        lastUpdateTime = System.currentTimeMillis();
```
Comment (qidaye) [inline fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalCatalog.java:282]: What's the difference?

File: fe/fe-core/src/main/java/org/apache/doris/datasource/EsExternalCatalog.java (hunk 1 above)
Comment (liugddx) [reply fe/fe-core/src/main/java/org/apache/doris/datasource/EsExternalCatalog.java:128]: > Why remove this?

I found that the logic of EsExternalCatalog is basically the same as that of the parent class, except that the data source is `default_db`. So I directly used the method of the parent class and it works.

File: fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalCatalog.java (hunk 2 above)
Comment (liugddx) [reply fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalCatalog.java:282]: > What's the difference?

This was just a redundant conversion so I removed it

**Review by morningman — DISMISSED** [review]
LGTM

**Review by qidaye — DISMISSED** [review]
LGTM

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
run buildall

**Review by qidaye — APPROVED** [review]
LGTM

**Comment by liugddx** [comment]
@dataroaring PTAL

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**